grub-efi/docker-create-grub-efi-binaries

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
I noticed that the original script was for `sh`. Is there a reason for it?

Reply:
Yes, it's because the `sh` in use in this container doesn't support `source`. It probably supports `.`, but I think `source` is better style.

Whether or not `source` is supported also seems to be inconsistent. Dash, which claims to stick to POSIX, does support it, so I'm not sure why the mentioned `sh` doesn't. Since this is only a build script, I didn't spend too much time investigating it.

Reply:
On second thought, I discovered that using `source` did not work in Yocto's shell either. Maybe I'll just switch back to `.` and put this back to what it was.

Reply:
Alright, I reverted this.

Reply:
Ok. I thought `source` is a built-in alias of bash for `.`. One of these days I have to read the POSIX standard ;)

Reply:
I thought so too. :stuck_out_tongue: